src/commands/supplement/passthrough/write.ts

```diff
@@ -15,7 +15,7 @@ export default class WritePassthrough extends Command {
     static flags = {
       help: Flags.help({char: 'h'}),
       input: Flags.string({char: 'i', required: true, description: 'An input Heimdall Data Format file'}),
-      passthroughFile: Flags.string({char: 'f', exclusive: ['passthroughData'], description: 'An input passthrough-data file (can contain any valid JSON); this flag or `passthroughData` must be provided'}),
+      passthroughFile: Flags.string({char: 'f', exclusive: ['passthroughData'], description: 'An input passthrough-data file (can contain any valid JSON); this flag or `passthroughData` must be provided \n\nPassthrough data can be any context/structure. See sample ideas on https://github.com/mitre/saf/wiki.\n\nA sample passthrough json used to provide CDM context could be something like this:\n	{"CDM":\n		{\n		"HWAM": {\n			"Asset_ID_Tattoo": "arn:aws:ec2:us-east-1:123456789012:instance/i-12345acbd5678efgh90",\n			"Data_Center_ID": "1234-5678-ABCD-1BB1-CC12DD34EE56FF78",\n			"FQDN": "i-12345acbd5678efgh90.ec2.internal",\n			"Hostname": "i-12345acbd5678efgh90",\n			"ipv4": "10.0.1.25",\n			"ipv6": "none defined",\n			"mac": "02:32:fd:e3:68:a1",\n			"os": "Linux",\n			"FISMA_ID": "ABCD2C21-7781-92AA-F126-FF987CZZZZ"\n			},\n		"CSM": {\n			"Server_Type": "member server",\n			"source_tool": "InSpec"\n			}\n		}\n	}'}),
```

Comment:
This comment also applies to target.

I don't think we should embed the passthrough sample within this flag, cause it's not DRY since we really ought to be putting it in the passthroughData field as well.  If you don't want to leave it only in the wiki, then IMO this information would be better in the description for the subcommand as a whole (line 8).  We could probably use the `summary` variable to have the shortened set of information and then use `description` for the sample json.

https://oclif.io/docs/commands

Reply:
I was hoping to find a different variable that could take a multi-line string. I tried summary I think but it didn't show up on --help

Reply:
Oh also, we need to actually make that wiki page with more samples.

Reply:
made changes according to conversation with eugene